ui/raidboss/timeline.js

```diff
@@ -43,11 +43,11 @@ class Timeline {
       return text;
 
     let orig = text;
-    let locale = this.options.Language || 'en';
+    let parserLang = this.options.PraserLanguage || 'en';
     if (replaceKey === 'replaceText')
-      locale = this.options.TimelineLanguage || this.options.Language || 'en';
+      parserLang = this.options.TimelineLanguage || this.options.PraserLanguage || 'en';
```

Comment:
I wonder a little if the default for TimelineLanguage and AlertsLanguage should be DisplayLanguage and not ParserLanguage.  What do you think?

Reply:
TimelineLanguage > ParserLanguage > 'en'
AlertsLanguage > DisplayLanguage > 'en'
looks good for me, as I said before, 
> Skill names are like "proper noun" thing.

Timeline contains mostly skill names, so it's better to be same as client's language.
However, Alerts doesn't, so display language would be better.